test/index.js

```diff
@@ -197,6 +205,157 @@ test('unknown element type', t => {
   t.deepEqual(apn.components, []);
 });
 
+test('embeds', t => {
+  const data = {
+    title: 'embeds',
+    body: [
+      {
+        type: 'embed',
+        embedType: 'instagram',
+        id: 'BDvcE47g6Ed',
+        caption: [
+          { type: 'text', href: 'http://mic.com', content: 'link' },
+          { type: 'linebreak' },
+          { type: 'text', content: 'normal text ' },
+          { type: 'text', bold: true, content: 'bold text ' },
+          { type: 'text', italic: true, content: 'italic text ' },
+          { type: 'text', bold: true, italic: true, content: 'bold italic text ' },
+          { type: 'text', mark: true, content: 'marked text' },
+          { type: 'text', mark: true, markClass: 'marker1' }
+        ]
+      },
+      {
+        type: 'embed',
+        embedType: 'twitter',
+        url: 'https://twitter.com/randal_olson/status/709090467821064196',
+        caption: [
+          { type: 'text', href: 'http://mic.com', content: 'link' },
+          { type: 'linebreak' },
+          { type: 'text', content: 'normal text ' },
+          { type: 'text', bold: true, content: 'bold text ' },
+          { type: 'text', italic: true, content: 'italic text ' },
+          { type: 'text', bold: true, italic: true, content: 'bold italic text ' },
+          { type: 'text', mark: true, content: 'marked text' },
+          { type: 'text', mark: true, markClass: 'marker1' }
+        ]
+      },
+      {
+        type: 'embed',
+        embedType: 'youtube',
+        youtubeId: 'oo6D4MXrJ5c',
+        caption: [
+          { type: 'text', href: 'http://mic.com', content: 'link' },
+          { type: 'linebreak' },
+          { type: 'text', content: 'normal text ' },
+          { type: 'text', bold: true, content: 'bold text ' },
+          { type: 'text', italic: true, content: 'italic text ' },
+          { type: 'text', bold: true, italic: true, content: 'bold italic text ' },
+          { type: 'text', mark: true, content: 'marked text' },
+          { type: 'text', mark: true, markClass: 'marker1' }
+        ]
+      },
+      {
+        type: 'embed',
+        embedType: 'image',
+        url: 'bundle://image.jpg',
```

Comment:
I think this module can produce a list of URLs to bundle
```
{
  "image.jpg": "https://thumbs.mic.com/abcde.jpg"
}
```
then https://github.com/micnews/apple-news can download them and upload to apple news.